Arrays.java

```diff
@@ -0,0 +1,69 @@
+public class Arrays {
+    public static void main(String[] args) {
+        int[] arrInt = new int[]{1, 2, 3};
+        double[] arrDouble = new double[]{1.57, 7.654, 9.986};
```

Comment:
```suggestion
        double[] arrDouble = {1.57, 7.654, 9.986};
```
можно короче написать